scripts/helpers.sh

```diff
@@ -13,6 +13,16 @@ is_osx() {
 	[ $(uname) == "Darwin" ]
 }
 
+is_chrome() {
+  chrome="/home/.shadow*"
+  path=$(df -Phk $HOME/Downloads | tail -1 | awk '{print $1}')
+  if [[ $path == $chrome ]]; then
+    return 0
+  else
+    return 1
+  fi
+}
+
```

Comment:
I do not own a Chromebook myself, but this seems like a very strange way of checking if one is using a Chromebook. Is it possible to use `uname` for checking if you're using a Chromebook?